homeassistant/components/vicare/water_heater.py

```diff
@@ -20,14 +21,16 @@
 VICARE_MODE_FORCEDREDUCED = "forcedReduced"
 VICARE_MODE_FORCEDNORMAL = "forcedNormal"
 VICARE_MODE_OFF = "standby"
+VICARE_MODE_ONETIMECHARGE =  "onetimecharge"
```

Comment:
Please separate words with underscore in variable names and strings.

Reply:
@MartinHjelmare I am happy to that however I was trying to be consistent with what exists. FORCEDNORMAL/FORCEDREDUCED are also supposed to be two words (and hence I just followed the standards for the component). If you want I can for sure change "onetimecharge" to "oneTimeCharge"?